applications/dashboard/views/settings/avatars.php

```diff
@@ -1,14 +1,45 @@
 <?php if (!defined('APPLICATION')) exit();
+/** @var Gdn_Form $form */
+$form = $this->Form;
 echo heading(t('Avatars')); ?>
-<div class="avatar-default">
+<?php foreach ($this->data('messages', []) as $message) : ?>
+    <div class="alert alert-info padded">
+        <?php echo $message; ?>
+    </div>
+<?php endforeach;
+
+$permissions = ['Garden.ProfilePicture.Edit', 'Garden.Profiles.Edit'];
+$permissions = implode(' or ', $permissions);
+$desc = 'Allow users with the '.$permissions.' permission to change their own avatars from their profile pages in Vanilla.';
+$allowEditPhotos = c('Garden.Profile.EditPhotos');
```

Comment:
We default this to `true` everywhere else, so this default is reversing the setting when no preference is set in the config yet.